pkg/builder/builder.go

```diff
@@ -62,6 +62,8 @@ const (
 	NMAKeyEnv             = "NMA_KEY_PATH"
 	NMASecretNamespaceEnv = "NMA_SECRET_NAMESPACE"
 	NMASecretNameEnv      = "NMA_SECRET_NAME"
+
+	HTTPServerVersionPath = "/v1/version"
```

Comment:
We probably don't need to export this. Suggest a rename:
```suggestion
	httpServerVersionPath = "/v1/version"
```